### DISCUSSION TITLE
Popper to place elements relative to the screen?

### DISCUSSION THREAD

**Discussion by marcusx2** [floating-ui/floating-ui#1239] [Q&A]
Can I use this library simply to place elements relative to the top of the screen, bottom of the screen, top right corner, top left corner, etc?

**Comment by FezVrasta** [comment]
Would a fixed positioned div be a better solution?

**Comment by marcusx2** [comment]
I want to be able to position everything without CSS. Can I use something like
```
Popper.createPopper(button, screen, {
    placement: 'right',
  });
```

So I can place something on the screen, to the right, left, top, bottom, etc?

**Comment by FezVrasta** [comment]
Take a look at this issue, there's an example to accomplish what you are looking for.

https://github.com/popperjs/popper-core/issues/965
